enhancements/ibm-public-cloud/ibm-public-cloud-support.md

```diff
@@ -0,0 +1,251 @@
+---
+title: IBM Public Cloud Support
+authors:
+  - "@csrwng"
+reviewers:
+  - "@derekwaynecarr"
+  - "@sttts"
+  - "@deads2k"
+  - "@mfojtik"
+  - "@ironcladlou"
+  - "@crawford"
+  - "@abhinavdahiya"
+  - "@spadgett"
+  - "@miabbott"
+approvers:
+  - "@derekwaynecarr"
+creation-date: 2020-02-04
+last-updated: 2020-02-04
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# IBM Public Cloud Support
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+IBM Public Cloud deploys Kubernetes/OpenShift clusters by hosting their control plane
+as pods in a central management cluster. OpenShift v3.11 is currently offered as a managed
+service using this model. This type of setup allows the hosting of many
+cluster control planes on a single Kubernetes cluster. Only the worker nodes are exclusive
+to an individual user cluster and may exist in a different account than the central
+management cluster.
+
+Supporting this type of deployment in OpenShift 4.x requires a different approach to
+installing OpenShift and managing its control plane. Instead of bootstrapping a cluster 
+control plane from scratch on virtual machines, a cluster control plane can be created 
+by applying a set of manifests to an existing cluster in a new namespace. 
+
+Changes are required in different areas of the product in order to make clusters deployed 
+using this method viable. These include changes to the cluster version operator (CVO), web 
+console, second level operators (SLOs) deployed by the CVO, and RHCOS.
```

Comment:
please don't use SLO as an acronym here. It is very commonly used for "Service Level Objectives".

Reply:
ack, will do a follow-up to remove.